modules/fedimint-mint-common/src/lib.rs

```diff
@@ -269,4 +269,6 @@ pub enum MintOutputError {
     InvalidAmountTier(Amount),
     #[error("The mint output version is not supported by this federation")]
     UnknownOutputVariant(#[from] UnknownMintOutputVariantError),
+    #[error("The mint output blind nonce was already used before")]
+    BlindNonceAlreadyUsed,
```

Comment:
New variant of the error. In practice it shouldn't happen, but when it does `SUBMIT_TRANSACTION_ENDPOINT.` on the client side will get confused.

Old clients will not understand it and probably fail to deserialize.

New clients in partially upgraded federation will not get `request_current_consensus_retry` and keep retrying.

Still better than burning money though. :shrug: